app/controllers/AuthenticationController.scala

```diff
@@ -429,6 +453,58 @@ class AuthenticationController @Inject()(
     }
   }
 
+  lazy val absoluteCallbackURL = s"${conf.Http.uri}/api/auth/oidc/callback"
```

Comment:
```suggestion
  lazy val absoluteOpenIdConnectCallbackUrl = s"${conf.Http.uri}/api/auth/oidc/callback"
```
(if this stays in the controller, I’d prefer it be named explicitly to more quickly know what it is about)